app/src/main/java/com/example/navjit/konnect/activity/MainActivity.java

```diff
@@ -338,7 +339,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.sign_out_menu:
                 mFirebaseAuth.signOut();
-                Auth.GoogleSignInApi.signOut(mGoogleApiClient);
+                SharedPreferences.Editor editor = userPreferences.edit();
+                editor.remove("loggedInUser");
+                editor.apply();
                 mUsername = ANONYMOUS;
```

Comment:
For the next commits please make username as current logged in user's first name and last name.